pkg/minikube/node/start.go

```diff
@@ -856,14 +856,20 @@ func tryRegistry(r command.Runner, driverName, imageRepository, ip string) {
 	}
```

Comment:
we should come up with a Curl equivalent of Windows command if they run it in powershell